Annex60/Fluid/Movers/BaseClasses/FlowMachineInterface.mo

```diff
@@ -469,112 +495,122 @@ the simulation stops.");
+                                                      delta=delta, d=preDer1, cBar=cBar, kRes=kRes),
+                            simplified=r_N*
                                 (cha.pressure(per=pCur1,
-                                                    V_flow=(1+delta)*V_flow_nominal, r_N=1,
-                                                    VDelta_flow=VDelta_flow, dpDelta=dpDelta,
-                                                    V_flow_max=V_flow_max, dpMax=dpMax,
-                                                    delta=delta, d=preDer1, cBar=cBar, kRes=kRes)
-                                -cha.pressure(per=pCur1,
-                                                    V_flow=(1-delta)*V_flow_nominal, r_N=1,
-                                                    VDelta_flow=VDelta_flow, dpDelta=dpDelta,
-                                                    V_flow_max=V_flow_max, dpMax=dpMax,
-                                                    delta=delta, d=preDer1, cBar=cBar, kRes=kRes))
-                                 /(2*delta*V_flow_nominal)));
-
-     else
-       dp = cha.pressure(per=pCur1, V_flow=V_flow, r_N=r_N,
-                                                VDelta_flow=VDelta_flow, dpDelta=dpDelta, V_flow_max=V_flow_max, dpMax=dpMax,
-                                                delta=delta, d=preDer1, cBar=cBar, kRes=kRes);
-     end if;
-     // end of computation for this branch
-   elseif (curve == 2) then
-    if homotopyInitialization then
-       dp = homotopy(actual=cha.pressure(per=pCur2, V_flow=V_flow, r_N=r_N,
-                                                    VDelta_flow=VDelta_flow, dpDelta=dpDelta,
-                                                    V_flow_max=V_flow_max, dpMax=dpMax,
-                                                    delta=delta, d=preDer2, cBar=cBar, kRes=kRes),
-                          simplified=r_N*
-                              (cha.pressure(per=pCur2,
-                                                    V_flow=V_flow_nominal, r_N=1,
-                                                    VDelta_flow=VDelta_flow, dpDelta=dpDelta,
-                                                    V_flow_max=V_flow_max, dpMax=dpMax,
-                                                    delta=delta, d=preDer2, cBar=cBar, kRes=kRes)
-                               +(V_flow-V_flow_nominal)*
+                                                      V_flow=V_flow_nominal, r_N=1,
+                                                      VDelta_flow=VDelta_flow, dpDelta=dpDelta,
+                                                      V_flow_max=V_flow_max, dpMax=dpMax,
+                                                      delta=delta, d=preDer1, cBar=cBar, kRes=kRes)
+                                 +(V_flow-V_flow_nominal)*
+                                  (cha.pressure(per=pCur1,
+                                                      V_flow=(1+delta)*V_flow_nominal, r_N=1,
```

Comment:
Can you make a smaller diff? I have a hard time checking what changed. For example, it used to be (1-delta) but now it is (1+delta)? This is probably a bug?